charts/vh-video-web/values.dev.template.yaml

```diff
@@ -3,6 +3,7 @@ java:
   image: "${IMAGE_NAME}"
   ingressHost: ${SERVICE_FQDN}
   releaseNameOverride: ${RELEASE_NAME}
+  disableTraefikTls: false
```

Comment:
just for info, this is needed to fix a problem with the PR instance of the app not working. The same change is being made to admin web